images/installer/root/etc/inventory-generator-config.yaml

```diff
@@ -0,0 +1,23 @@
+---
+# meta config
+master_config_path: null  # defaults to /opt/app-root/src/master-config.yaml
+admin_kubeconfig_path: null  # defaults to /opt/app-root/src/.kube/config
```

Comment:
How helpful is it to include these? 

Feels like the default paths are repeated in many places.